dna/ios/JWPlayer/JWPlayerSample/JWPlayerQosModule.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  JWPlayerQosModule.swift
+//  JWPlayerSample
+//
+//  Created by Boris BORGOBELLO on 04/09/2020.
+//  Copyright © 2020 Streamroot. All rights reserved.
+//
+
+// DO NOT MODIFY //
+
+import Foundation
+import StreamrootSDK
+import JWPlayer_iOS_SDK
+
+class JWPlayerQosModule : NSObject, JWPlayerDelegate {
```

Comment:
Any possibility to integrate `updateDroppedFrameCount` and `trackSwitchOccurred` with JWPlayer?